packages/trader/src/AppV2/Containers/Trade/trade-types.tsx

```diff
@@ -125,6 +126,24 @@ const TradeTypes = ({ contract_type, onTradeTypeSelect, trade_types }: TTradeTyp
         }));
     };
 
+    const scrollToSelectedTradeType = () => {
+        setTimeout(() => {
+            let position_x = 0;
+            if (trade_types_ref?.current) {
+                const selected_chip = trade_types_ref?.current.querySelector(
+                    'button[data-state="selected"]'
+                ) as HTMLButtonElement;
+                if (selected_chip) {
+                    position_x = selected_chip?.getBoundingClientRect()?.x || 0;
+                }
+                trade_types_ref.current.scrollBy({
+                    left: position_x - 16,
```

Comment:
if there is no selected_chip, the position_x will be 0 and the left might have the value as -16?
Can we prevent that?